src/main.ts

```diff
@@ -10,6 +11,23 @@ var azPSHostEnv = !!process.env.AZUREPS_HOST_ENVIRONMENT ? `${process.env.AZUREP
 
 async function main() {
     try {
+        //Options for error handling
+        let commandStdErr = false;
+        const loginOptions: ExecOptions = {
+            silent: true,
+            ignoreReturnCode: true,
+            failOnStdErr: true,
+            listeners: {
+                stderr: (data: Buffer) => {
+                    let error = data.toString().replace('ERROR','');
```

Comment:
we want to replace only the starting ERROR substring and not all ERROR mentions right?

Reply:
Made small change. It replaces only first occurrence in the beginning of the message  now.